openmetadata-ui/src/main/resources/ui/src/components/Alerts/AlertDetails/AlertDiagnosticInfo/alert-diagnostic-info-tab.less

```diff
@@ -0,0 +1,58 @@
+/*
+ *  Copyright 2025 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+@import (reference) url('../../../../styles/variables.less');
+
+.alert-diagnostic-container {
+  .ant-card {
+    border-radius: 8px;
+  }
+
+  .ant-statistic {
+    .ant-statistic-title {
+      color: @primary-color;
+      font-size: 14px;
+    }
+
+    .ant-statistic-content {
+      font-size: 24px;
+      font-weight: 600;
+    }
+  }
+
+  .ant-typography {
+    margin-bottom: 16px;
+  }
+
+  .text-grey-muted {
+    color: @primary-color;
+  }
```

Comment:
`text-grey-muted` is already defined in `font.less`. Also, I am not sure why we are giving primary color for text-grey-muted class